Gemfile.lock

```diff
@@ -266,47 +266,48 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
```

Comment:
### [aws-sigv4](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.6.1 → 1.8.0

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/version-3/gems/aws-sigv4/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/version-3/gems/aws-sigv4), [gem-diff](https://my.diffend.io/gems/aws-sigv4/1.6.1/1.8.0)_]





<details>
<summary>Commits</summary>

A change of **221** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.6.1...1.8.0).

These are the first 10 commits:
- (6cf088f) [normalize s3 key in delete xml](https://github.com/aws/aws-sdk-ruby/commit/6cf088f28741cc57dd89d1700770c8c16a583bbf)
- (9776a57) [Fix issue breaking enumerable#include? in AWS::EC2::Security…](https://github.com/aws/aws-sdk-ruby/commit/9776a574707e1271f91f5001bc42bcae97e45745)
- (53d960f) [Version 1.6.2 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/53d960fc2eabae3d592fa78fbcccbbf30a311587)
- (b91a8ba) [Merge pull request #64 from mrardon/master](https://github.com/aws/aws-sdk-ruby/commit/b91a8baf1572e6e25d98e102f5437416ab145fc1)
- (9877f13) [Version 1.6.3 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/9877f13179fdffaa31484416ecf690d52597a767)
- (8f92494) [spec: do not force simplecov presence.](https://github.com/aws/aws-sdk-ruby/commit/8f92494e3820fe38842869168d42bd14cd3c2678)
- (42c8bc8) [Allow for setting of Content-Disposition when performing cop…](https://github.com/aws/aws-sdk-ruby/commit/42c8bc80967b01afd73f6b92338482aa0b58668c)
- (61a5282) [Merge pull request #58 from mikewatt/normalize-xml-s3-key-na…](https://github.com/aws/aws-sdk-ruby/commit/61a5282cde2762f12024c553a789cf489dcc6ee8)
- (302bfac) [Merge pull request #66 from moorecp/master](https://github.com/aws/aws-sdk-ruby/commit/302bfac312c2583fc5bba0b825ba88b9ef6706a5)
- (711a098) [Merge pull request #65 from gentoo/master](https://github.com/aws/aws-sdk-ruby/commit/711a0988925683e033ac767e56bfc5ba5ec256f3)


</details>